INSTALL.md

```diff
@@ -119,6 +119,16 @@ export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$(brew --prefix)/opt/libpq/lib/pkgconfi
 export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$(brew --prefix)/opt/openssl@3/lib/pkgconfig"
 export PATH="$(brew --prefix bison)/bin:$PATH"
 ```
+- Install `clang-format-12` 
+> Note: `brew` does not contain a cask for `clang-format-12`.
+
+To install `clang-format-12`:
+1. Download [clang+llvm-12.0.0-x86_64-darwin.tar.xz](https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/clang+llvm-12.0.0-x86_64-apple-darwin.tar.xz)
```

Comment:
Instead of descriptions with example commands, it might be easier just to have a block that people can just copy paste. You essentially already have that, just turn the first download step into a `curl` and get rid of the words.